viziphant/rasterplot.py

```diff
@@ -178,13 +180,6 @@ def rasterplot(spiketrain_list,
     :param palette: string | sequence
         Define the color palette either by its name or use a custom palette in
         a sequence of the form ([r,g,b],[r,g,b],...).
-    :param colorcodes: str
```

Comment:
@rgutzen Can you please comment -- should the comment be removed as suggested here, or the option added?